### PR TITLE
Milestone2 components

### PR DESCRIPTION
**Pull request by RaminMammadzada**
- Created stateless App component.
- Created stateful ButtonPanel component and related buttons inside it.
- Created Button and Display components, needed statements to populate props.

### CODE REVIEW

File: src/components/ButtonPanel.js
Comment (Torres-ssf) [inline src/components/ButtonPanel.js]: - [x] We can remove the `+` sign from the number 5

File: src/components/Display.js
Comment (Torres-ssf) [inline src/components/Display.js]: - [ ] Since the proptype is defined as a String, it needs to make the default value as a string, not a number
```JS
Display.defaultProps = {
  result: '0',
};

```

File: src/components/Button.js
Comment (Torres-ssf) [inline src/components/Button.js]: OPTIONAL

- [x] Since we do not have any case of a button without a name, it makes sense to make the `buttonName` props required, therefore, a default value is not required.
```JS
Button.propTypes = {
  buttonName: PropTypes.string.isRequired,
};

```

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :clap: 
Hi @RaminMammadzada 
Awesome job on this milestone so far.
But before moving on we need to make minor changes.

- [ ] Check my comments below

Happy coding :computer: 
**Sérgio Torres**

File: src/components/ButtonPanel.js
Comment (RaminMammadzada) [reply src/components/ButtonPanel.js]: Done.

File: src/components/Display.js
Comment (RaminMammadzada) [reply src/components/Display.js]: As you see in the code, `'0'` is a string not a number.

File: src/components/Button.js
Comment (RaminMammadzada) [reply src/components/Button.js]: It is good advice, thank you. Done.

File: src/components/Display.js
Comment (teekaytech) [inline src/components/Display.js]: Your result property still carries the value of the number `0`. As mentioned by the previous reviewer, this value should be a string (i.e. `result: '0'`) and not a number (i.e. `result: 0`) as you have in your source code.
[(should be String screenshot)](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))

File: src/components/ButtonPanel.js
Comment (teekaytech) [inline src/components/ButtonPanel.js]: The '+' symbol. still exists in the '5' button. Kindly remove it.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__Thank you for implementing some of the changes required by the last reviewer.
However, some changes are required before this milestone can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] Check my comments below and make changes as appropriate.

**[ OPTIONAL CHANGES ]**

- [ ] Kindly use ES6 syntax (e.g. arrow functions for functional components) in your application.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****


 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**
